edvart/report.py

```diff
@@ -119,13 +127,7 @@ def _generate_notebook(
         )
 
         # Add imports cell
-        imports_set = {
-            "import pandas as pd",
-            "import os",
-            "from typing import Any, Callable, Dict, List, Optional, Tuple, Union",
-            "import plotly.offline as py",
-            "import plotly.io as pio",
-        }
+        imports_set = self._DEFAULT_IMPORTS
         if extra_imports is not None:
             imports_set.update(extra_imports)
```

Comment:
Wouldn't this modify `_DEFAULT_IMPORTS` for all future reports? I think we need to use deepcopy:
```
import copy

imports_set = copy.deepcopy(self._DEFAULT_IMPORTS)
```

Reply:
You're right. Added coyping. Simple `copy` should be enough since it is a flat set.